src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -124,15 +123,13 @@ class BlockingProcess(
     try {
       val selfRef = context.self
 
-      val stdin = Sink.outputStream(process.getOutputStream)
+      val stdin = StreamConverters.fromOutputStream(process.getOutputStream)
         .mapMaterializedValue(_.andThen { case _ => selfRef ! StdinTerminated })
 
-      val stdout = Source.inputStream(process.getInputStream)
-        .withAttributes(Attributes.inputBuffer(1, 1))
```

Comment:
Removing `withAttributes` due to the fact that `Source` from `InputStream` now emits data eagerly: https://github.com/akka/akka/issues/19193